scripts/lib.sh

```diff
@@ -116,13 +116,11 @@ install-py-dev() {
     # Webpacked assets are only necessary for devserver (which does it lazily anyways)
     # and acceptance tests, which webpack automatically if run.
     SENTRY_LIGHT_BUILD=1 pip install -e '.[dev]'
-    patch-selenium
 }
 
 patch-selenium() {
     # XXX: getsentry repo calls this!
-    # This hack is until we can upgrade to a newer version of Selenium
-    python -S -m tools.patch_selenium
+    :
```

Comment:
planning to delete this later I assume?

Reply:
yeah i can't delete any of the hack (except this call) until after getsentry doesn't call it (annoyingly)